packages/components/addon/components/hds/table/table-sortable.js

```diff
@@ -1,26 +1,33 @@
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
 import { guidFor } from '@ember/object/internals';
+import { tracked } from '@glimmer/tracking';
 
 export default class HdsTableTableSortableComponent extends Component {
   a11yMessageId = 'table-a11y-message-' + guidFor(this);
 
-  get sort() {
-    return `${this.args.sortBy}:${this.args.sortOrder}`;
+  @tracked sortBy = this.args.sortBy;
+  @tracked sortOrder = this.args.sortOrder || 'asc';
+
+  get getSortCriteria() {
+    return `${this.sortBy}:${this.sortOrder}`;
   }
 
   @action
   setSortBy(column) {
-    if (this.args.sortBy === column) {
+    if (this.sortBy === column) {
       //invert the sort order
-      this.args.setSortOrder(this.args.sortOrder === 'asc' ? 'desc' : 'asc');
+      this.sortOrder = this.sortOrder === 'asc' ? 'desc' : 'asc';
     } else {
-      this.args.setSortBy(column);
-      this.args.setSortOrder('asc');
+      this.sortBy = column;
+      this.sortOrder = 'asc';
     }
 
+    const currentColHeading = event.target;
+    const currColHeadingText = currentColHeading.textContent;
+
     const liveRegion = document.querySelector(`#${this.a11yMessageId}`);
-    liveRegion.textContent = `Sorted by ${column} ${this.args.sortOrder}ending`;
+    liveRegion.textContent = `Sorted by ${currColHeadingText} ${this.sortOrder}ending`;
     setTimeout(function () {
       liveRegion.textContent = '';
     }, 2000);
```

Comment:
[issue] you're overriding some of the changes already approved and merged in https://github.com/hashicorp/design-system/pull/625

see: https://github.com/hashicorp/design-system/blob/cb4c1554a175cbf77dd1d547f76a93c1d8e6e7de/packages/components/addon/components/hds/table/table-sortable.js#L13-L26

I think you need to merge `table-sortsupport` in this branch, or rebase this branch on `table-sortsupport` but avoid to bring in these changes (I can help you with this, if needed)

Reply:
this section isn't working at all, working on it now.